Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN mkdir -p "${THEME_PATH}" && chmod -R g+w "${THEME_PATH}" \
 COPY planet/planet.ini /var/planet/
 COPY planet/theme "${THEME_PATH}"
 COPY --from=configbuilder /tmp/config.ini "${THEME_PATH}"/
```

Comment:
Please move the new template file in the root of planet/theme.  This copy statement is not needed since we copy these files with line 32.